monocle/worker.py

```diff
@@ -1264,9 +1269,33 @@ def normalize_gym(raw):
             'prestige': raw.gym_points,
```

Comment:
is that still provided or can it be removed?
note: if you remove it there may be places where it is accessed without checking the availability